### ISSUE TITLE
n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218d (NOCOMP baseline)

### ISSUE THREAD

**Issue by rosiealice** [NorESMhub/NorESM3_land_only_dev_simulations#5]
**Purpose:**
Baseline run of the present FATES-SP tag with MEGAN switched on, but without matvey's additional changes. 
Uses a slighlty modified parameter file to speed up the simulation by changing to determnistic competition. 
__
**Description:**

To get the code
> git clone https://github.com/NorESMhub/NorESM.git
> cd NorESM
> git checkout noresm2_5_alpha08d
> ./bin/git-fleximod update

- NOTE: a memory leak is present in the run. 
See https://github.com/NorESMhub/NorESM/issues/550

- **create_newcase commnad**
```
./create_newcase --case n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218d --compset 1850_DATM%QIA_CLM60%FATES_SICE_SOCN_MOSART_SGLC_SWAV --res ne30pg3_tn14 --project NN9188K --run-unsupported --mach betzy -pecount L
- **user_nl_clm**

fates_paramfile = '/cluster/work/users/rosief/git/calibration_parameter_files/SP_calib/fates_params_det_alpha09.nc'

use_fates_nocomp=TRUE
use_fates_fixed_biogeog=TRUE
hist_fexcl1 =
	      'FATES_VEGC_PF',
              'FATES_STOREC_PF', 'FATES_RECRUITMENT_PF', 'FATES_MORTALITY_PF', 'FATES_PATCHAREA_AP', 'FATES_LAI_AP',
              'FATES_CANOPYAREA_AP', 'FATES_NESTEROV_INDEX', 'FATES_IGNITIONS', 'FATES_FDI', 'FATES_ROS', 'FATES_EFFECT_WSPEED',
              'FATES_FUELCONSUMED', 'FATES_FIRE_INTENSITY', 'FATES_FIRE_INTENSITY_BURNFRAC', 'FATES_BURNFRAC', 'FATES_FUEL_MEF',
              'FATES_FUEL_BULKD', 'FATES_FUEL_EFF_MOIST', 'FATES_FUEL_SAV', 'FATES_FUEL_AMOUNT', 'FATES_FRAGMENTATION_SCALER_SL',
              'FATES_FUEL_MOISTURE_FC', 'FATES_FUEL_AMOUNT_FC', 'FATES_BURNFRAC_AP', 'FATES_FIRE_INTENSITY_BURNFRAC_AP',
              'FATES_FUEL_AMOUNT_AP', 'FATES_FUEL_BURNT_BURNFRAC_FC',
	      'FATES_LITTER_IN_EL', 'FATES_LITTER_OUT_EL', 'FATES_SEED_BANK_EL', 'FATES_SEEDS_IN_LOCAL_EL',
              'FATES_SEEDS_IN_EXTERN_EL', 'FATES_SEED_GERM_EL', 'FATES_SEED_DECAY_EL', 'FATES_STOREC', 'FATES_VEGC',
              'FATES_SAPWOODC', 'FATES_FROOTC', 'FATES_REPROC', 'FATES_STRUCTC', 'FATES_NONSTRUCTC', 
              'FATES_VEGC_ABOVEGROUND', 'FATES_CANOPY_VEGC', 'FATES_USTORY_VEGC', 'FATES_PRIMARY_PATCHFUSION_ERR',
              'FATES_DISTURBANCE_RATE_FIRE', 'FATES_DISTURBANCE_RATE_LOGGING', 'FATES_DISTURBANCE_RATE_TREEFALL',
              'FATES_HARVEST_WOODPROD_C_FLUX', 'FATES_GPP_CANOPY', 'FATES_AUTORESP_CANOPY',
              'FATES_GPP_USTORY', 'FATES_AUTORESP_USTORY', 'FATES_CROWNAREA_CL', 'FATES_DEMOTION_CARBONFLUX',
              'FATES_PROMOTION_CARBONFLUX', 'FATES_MORTALITY_CFLUX_CANOPY', 'FATES_MORTALITY_CFLUX_USTORY', 
              'FATES_DDBH_CANOPY_SZ', 'FATES_DDBH_USTORY_SZ', 'FATES_BASALAREA_SZ',
              'FATES_VEGC_ABOVEGROUND_SZ', 'FATES_LAI_CANOPY_SZ', 'FATES_MORTALITY_CANOPY_SZ', 'FATES_NPLANT_USTORY_SZ',
              'FATES_LAI_USTORY_SZ', 'FATES_NPLANT_SZ', 'FATES_NPLANT_AC', 'FATES_MORTALITY_BACKGROUND_SZ',
              'FATES_MORTALITY_HYDRAULIC_SZ', 'FATES_MORTALITY_CSTARV_SZ', 'FATES_MORTALITY_IMPACT_SZ',
              'FATES_MORTALITY_FIRE_SZ', 'FATES_MORTALITY_TERMINATION_SZ', 'FATES_MORTALITY_LOGGING_SZ',
              'FATES_MORTALITY_FREEZING_SZ', 'FATES_MORTALITY_SENESCENCE_SZ', 'FATES_MORTALITY_AGESCEN_SZ',
              'FATES_MORTALITY_AGESCEN_AC', 'FATES_MORTALITY_USTORY_SZ', 'FATES_FROOTMAINTAR', 'FATES_CROOTMAINTAR',
              'FATES_LSTEMMAINTAR', 'FATES_NEP', 'FATES_HET_RESP', 'FATES_FIRE_CLOSS', 'FATES_FIRE_FLUX_EL',
              'FATES_LITTER_AG_FINE_EL', 'FATES_LITTER_BG_FINE_EL', 'FATES_LITTER_BG_CWD_EL', 'FATES_LITTER_AG_CWD_EL',
              'FATES_LITTER_CWD_ELDC',
'FATES_NPP_PF','FATES_MORTALITY_LOGGING_SE_SZ',
'FATES_MORTALITY_HYDRAULIC_SE_SZ','FATES_MORTALITY_FREEZING_SE_SZ',
'FATES_MORTALITY_CSTARV_SE_SZ','FATES_MORTALITY_CANOPY_SE_SZ',
'FATES_MORTALITY_CANOPY_SE_SZ','FATES_MORTALITY_BACKGROUND_SE_SZ',
'FATES_MORTALITY_BACKGROUND_SE_SZ','FATES_MORTALITY_AGESCEN_SE_SZ',
'FATES_VEGC_SE_PF','FATES_RECRUITMENT_CFLUX_PF','FATES_NPP_SE_PF',
'FATES_NPP_SE_PF','FATES_NPLANT_SEC_PF','FATES_NOCOMP_PATCHAREA_PF',
'FATES_NOCOMP_NPATCHES_PF','FATES_NOCOMP_BURNEDAREA_PF',
'FATES_MORT_CSTARV_CONT_CFLUX_PF','FATES_MORTALITY_HYDRO_CFLUX_PF',
'FATES_MORTALITY_FIRE_CFLUX_PF','FATES_MORTALITY_CSTARV_CFLUX_PF',
'FATES_MORTALITY_CFLUX_PF','FATES_MEANSMP_DROUGHTPHEN_PF',
'FATES_MEANLIQVOL_DROUGHTPHEN_PF','FATES_LEAFC_PF',
'FATES_L2FR_USTORY_REC_PF','FATES_L2FR_CANOPY_REC_PF',
'FATES_GPP_SE_PF','FATES_ELONG_FACTOR_PF','FATES_DROUGHT_STATUS_PF',
'FATES_DAYSINCE_DROUGHTLEAFON_PF','FATES_DAYSINCE_DROUGHTLEAFOFF_PF',
'FATES_CANOPYCROWNAREA_PF','FATES_TRANSITION_MATRIX_LULU',
'FATES_DISTURBANCE_RATE_MATRIX_LULU','FATES_PATCHAREA_LU',;
'
```
- **xmlchange commands**
```
./xmlchange STOP_OPTION=nyears
./xmlchange STOP_N=2
./xmlchange RESUBNIT=20


```
___
**Compset long name**
1850_DATM%QIA_CLM60%FATES_SICE_SOCN_MOSART_SGLC_SWAV
___
**Model grid long name**
`a%ne30np4.pg3_l%ne30pg4.pg3_oi%tnx1v4_r%r05_w%null_z%null_g%gris4_m%tnx1v4`

**Case directory:**
- Locally on betzy: 
   `/cluster/work/users/rosief/git/noresm2_5_alpha08d/cime/scripts/n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241212

- On github:
tbd
___
**Code version on github (either NorESMhub or user's fork):**
- On github:  https://github.com/NorESMhub/NorESM/releases/tag/noresm2_5_alpha08d
- https://github.com/username/noresm/tree/<code_version>
___
**Diagnostics:**
- Diags (if available):
https://ns9560k.web.sigma2.no/datalake/diagnostics/noresm/rosief/n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c/

TL:DR This run has a low LAI and lots of patches... 
___
**Output:**
- Raw output on nird (if still available): 
/datalake/NS9560K/noresm3/cases/n1850.FATES-SP.ne30_tn14.alpha08d.20241209/lnd/hist/ 
___
**Contacts:**
@rosiealice 
___
**Extra details:**
